src/Autocomplete/Autocomplete.jsx

```diff
@@ -193,7 +196,8 @@ Autocomplete.defaultProps = {
     value ? (Array.isArray(value) ? value.includes(v) : value === v) : false,
   onChange: () => console.warn('Autocomplete.onChange should be a function'),
   placeholder: 'Please select an option',
-  searchMaxResults: 30
+  searchMaxResults: 30,
+  optionalText: 'Optional'

```

Comment:
how does this work with the localization requirements ?

Reply:
In en/de the word is the same, but I intend to provide this prop when using the component.